var/spack/repos/builtin/packages/callflow/package.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+import os
+import stat
+from spack import *
+
+
+class Callflow(PythonPackage):
+    """CallFlow is an interactive visual analysis tool that provides a
+       high-level overview of CCTs together with semantic refinement
+       operations to progressively explore the CCTs."""
+
+    homepage = "https://github.com/LLNL/CallFlow"
+    url      = "https://github.com/LLNL/CallFlow/archive/v1.1.0.tar.gz"
+    git      = 'git@github.com:LLNL/CallFlow.git'
+
+    maintainers = ["bhatiaharsh", "jarusified"]
+
+    version('1.1.0', sha256='f8b875eb62fbac04b117e3c23fccff99d768158226a9b7fa222a2b2a6acafa44')
+    version('v1.1.1', tag='v1.1.1')
```

Comment:
Why is one version downloaded from a checksum and the other from a git tag? Why is one version prefixed with `v` and the other isn't? You should also sort versions from newest to oldest.